src/interface/src/app/shared/animations.ts

```diff
@@ -51,3 +54,35 @@ export const opacityTransitionTrigger = (params: {
     transition('opaque => transparent', [animate(params.timingA)]),
     transition('transparent => opaque', [animate(params.timingB)]),
   ]);
+
+export const triggerAnimation = trigger('expandCollapsePanel', [
```

Comment:
Could we call this something else? "triggerAnimation" is a bit nonspecific

Reply:
Indeed it is. done!